api-spec.json

```diff
@@ -0,0 +1,15640 @@
+{
+   "oappy": "0.0.1",
+   "info": {
+      "name": "enigma",
+      "go-package-name": "enigma",
+      "go-package-import-path": "github.com/qlik-oss/enigma-go",
+      "version": "devbuild",
+      "description": "enigma-go is a library that helps you communicate with a Qlik Associative Engine.",
+      "license": "MIT",
+      "x-qlik-visibility": "public",
+      "x-qlik-stability": "locked"
+   },
+   "definitions": {
+      "AlfaNumString": {
+         "deprecated": true,
+         "x-qlik-deprecation-description": "This will be removed in a future version",
+         "type": "struct",
+         "entries": {
+            "IsNum": {
+               "description": "Is set to true if the value is a numeric.",
+               "type": "bool"
+            },
+            "String": {
+               "description": "Calculated value.",
+               "type": "string"
+            }
+         }
+      },
+      "AlternateStateData": {
+         "type": "struct",
+         "entries": {
+            "FieldItems": {
+               "description": "List of the selections.",
+               "type": "slice",
+               "items": {
+                  "type": "enigma.BookmarkFieldItem"
```

Comment:
This is a ref to another object in this json, can we mark this in any way or use standard json refs e.g. "#/definitions/BookmarkFieldItem"